.gitignore

```diff
@@ -44,7 +44,7 @@ incoming_chat.txt
 locobot/test/test_assets/perception_handlers/
 locobot/agent/dashboard_data.db*
 nsp_outputs.csv
-agents/locobot/interaction_loggings.json
+agents/locobot/interaction_loggings_*
```

Comment:
still think you should add paths like *.csv, *.json, or *.log, because other components can write logs too that you don't want committed